### PR TITLE
Pre launch changes 1

### PR DESCRIPTION
**Pull request by BrewertonSantos**
**Reviewing changes**

- variable calls were created for the items below in the InsertAllInputs method in the ModelConfig.css file;
- density applied to the lines (119, 387) from the entry made by the user in the calculatorform;
- mass applied to the lines (120, 388) from the entry made by the user in the calculatorform;
- aerodynamic coefficient applied to the lines (127, 396) from the entry made by the user in the calculatorform;
- updated fixed directory for static program standard defined in variable DirectoryDialog.defaultRepo;
- moved the Compile method from the ModelConfig.cs file to the PromptEvents.cs file
- removed folder variable in ShowFileContent method;
- updated fixed to static directory in the Actions.cs file ShowFileContent method;
- unused libraries in files are removed:
    Default.xaml.cs;
    CalculatorForm.xaml.cs.
- Created console log (Console.xaml).

### CODE REVIEW

File: WindowsClient/_Data/_Actions/Actions.cs [hunk 1]
```diff
@@ -18,10 +18,8 @@ class Actions
 
         public static void ShowFileContent(string fileName)
         {
-            string folder = Path.Combine(Environment.GetFolderPath(
-    Environment.SpecialFolder.ApplicationData), ".PIM");
             // Filename  
-            string fullPath = folder + "/" + fileName;
+            string fullPath = DirectoryDialog.defaultRepo + "/" + fileName;
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/Actions.cs:22]: replace this line with $"{}" for future possible implementions and code readability

File: WindowsClient/_Data/_Actions/ModelConfig.cs [hunk 2]
```diff
@@ -14,9 +14,9 @@ class ModelConfig
         public static Process process = new Process();
         public static ProcessStartInfo Info = new ProcessStartInfo();
 
-        public static void InsertAllInputs(float lat1, float lat2, float time)
+        public static void InsertAllInputs(float iLat, float iLong, float iHeight, float fLat, float fLong, float fHeight, float time, float dens, float aeroDinCo, float mass, string tab)
         {
-            string folder = "C:/Users/BREWERTONTHIAGOOLIVE/Desktop";//_Data._Actions.DirectoryDialog.Path();
+            string folder = DirectoryDialog.defaultRepo;
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/ModelConfig.cs:19]: OK

File: WindowsClient/_Data/_Actions/ModelConfig.cs [hunk 3]
```diff
@@ -30,34 +30,21 @@ public static void InsertAllInputs(float lat1, float lat2, float time)
...
+            readLines[140 - 1] = $"      T = {time}d0"; // time between initial and final catches
+            readLines[144 - 1] = $"        H1={iHeight}d0"; // observation object initial height
+            readLines[145 - 1] = $"        LON1= {iLong}d0"; // observation object initial longitude
+            readLines[146 - 1] = $"        LAT1={iLat}d0"; // observation object initial latitude
+            readLines[148 - 1] = $"      H2={fHeight}d0"; // observation object final height
+            readLines[149 - 1] = $"        LON2={fLong}d0"; // observation object final longitude
+            readLines[150 - 1] = $"        LAT2={fLat}d0"; // observation object final latitude
+            readLines[387 - 1] = $"            densiMeteor={dens}"; // observation object density
+            readLines[388 - 1] = $"	    M = {mass}.d-3"; // observation object mass
+            readLines[396 - 1] = $"	    CD = {aeroDinCo}d0"; // aerodynamic coefficient
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/ModelConfig.cs:45]: what are those spaces for?

Comment (lucasgdb) [reply WindowsClient/_Data/_Actions/ModelConfig.cs:45]: maybe, use \t (tabulation) instead

File: WindowsClient/_Data/_Actions/ModelConfig.cs [hunk 4]
```diff
@@ -30,34 +30,21 @@ public static void InsertAllInputs(float lat1, float lat2, float time)
...
         }
-
-        public static void Compile(string directory, string compiler, string inputfilename, string outputfilename, string format)
-        {            Info.WindowStyle = ProcessWindowStyle.Hidden;
-            Info.FileName = "cmd.exe";
-            // if don't use [/C] don't work, u need add a directory string and input in Info.Arguments.
-            Info.Arguments = $"/C cd c:/Users/BREWERTONTHIAGOOLIVE/Desktop & {compiler} {inputfilename}.{format} -o {outputfilename}.{format}";
-            process.StartInfo = Info;
-            process.Start();
-        }
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/ModelConfig.cs:61]: OK

File: WindowsClient/_Data/_Items/Console.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace WindowsClient._Data._Items
+{
+    /// <summary>
+    /// Interaction logic for Console.xaml
+    /// </summary>
+    public partial class Console : UserControl
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Items/Console.xaml.cs:21]: what does this class do?

File: WindowsClient/_Data/_Actions/PromptEvents.cs [hunk 6]
```diff
@@ -17,7 +18,7 @@ public static void Compile(string directory, string compiler, string inputfilena
             Info.WindowStyle = ProcessWindowStyle.Hidden;
             Info.FileName = "cmd.exe";
             // if don't use [/C] don't work, u need add a directory string and input in Info.Arguments.
-            Info.Arguments = $"/C cd c:/Users/BREWERTONTHIAGOOLIVE/Desktop & {compiler} {inputfilename}.{format} -o {outputfilename}.{format}";
+            Info.Arguments = $"/C cd {DirectoryDialog.defaultRepo} & {compiler} {inputfilename}.{format} -o {outputfilename}.{format}";
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/PromptEvents.cs:21]: OK

File: WindowsClient/_Data/_Actions/DirectoryDialog.cs [hunk 7]
```diff
@@ -12,6 +12,7 @@ namespace WindowsClient._Data._Actions
     // Manipulate OpenFileDialog Property
     class DirectoryDialog
     {
+        public static string defaultRepo = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), ".PIM");
```
Comment (lucasgdb) [inline WindowsClient/_Data/_Actions/DirectoryDialog.cs:15]: OK

File: WindowsClient/_Data/_Items/Console.xaml.cs (hunk 5 above)
Comment (BrewertonSantos) [reply WindowsClient/_Data/_Items/Console.xaml.cs:21]: write log warnings in console, but it's not ready.

**Pull request merged**